torch/_prims/nvfuser_executor.py

```diff
@@ -268,6 +268,29 @@ def __call__(self, *args):
         )
 
 
+# A set of operators that are supported by nvFuser
+# but should not form a fusion group solely on their own
+# _non_compute_ops = {
+#     "torch.ops.nvprims.transpose.default",
+#     "torch.ops.nvprims.view_of.default",
+#     "torch.ops.nvprims.broadcast_in_dim.default",
+#     "torch.ops.nvprims.squeeze.default",
+# }
+_non_compute_ops = [
```

Comment:
nitpick: peeking through op return type and categorizing `view`-like return to non-compute ops is counter-intuitive. (thinking about in-place update).

I think this is safe for our uses, but should we rename this to `_view_like_ops` and add a line in the comment that `view-like ops are likely no-compute ops, since in-place update is handled by functionalization`

Reply:
I agree with you, I used the same name as was already used in the partitioner code.
We also don't have the in-place update (not merged yet).